### PR TITLE
test-requirements.txt: Change the requirement for pip

### PR DESCRIPTION
**Pull request by aadilmehdis**
Made changes to bar the pip versions 9.0.2 and 10.0
specifically from in the test-requirements.txt.

Fixes https://github.com/coala/coala-bears/issues/2404

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .pytest_cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (virresh) [inline .pytest_cache/v/cache/lastfailed]: Unrelated change
Please do not commit .pytest_cache
It seems you used `git add .`
Please keep in mind that these files are temporary files that should not be pushed
If you get stuck while amending your commit, consider asking your doubts on our [gitter](https://gitter.im/coala/coala-bears) channel :)

File: .pytest_cache/v/cache/nodeids [hunk 2]
```diff
@@ -0,0 +1,760 @@
+[
```
Comment (virresh) [inline .pytest_cache/v/cache/nodeids]: Please remove this file as well

Comment (aadilmehdis) [reply .pytest_cache/v/cache/nodeids]: Right, so I make a new commit?

**Comment by aadilmehdis** [comment]
Oh, I am sorry. I am new to git.
So should I close the previous pull requests?

**Comment by virresh** [comment]
Yes, It is a bad practice to close PRs though, but not much can be done in this case.
You may close all except one PR and amend the one that you keep open :)

Keep in mind that you don't need to close all of them, keep one PR open, and fix it.

**Comment by jayvdb** [comment]
@aadilmehdis, come onto http://coala.io/chat and ask for someone to help you clean this up.

File: .pytest_cache/v/cache/nodeids (hunk 2 above)
Comment (li-boxuan) [reply .pytest_cache/v/cache/nodeids]: Don't make a new commit. Amend this commit. Take a look at this guide: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#editing-commit-messages

**Comment by aadilmehdis** [comment]
@jayvdb 
Hi, could you tell me the issue with the pull request? (since its marked  process/wip )
I am not clear with it.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2497/commits/44df5a910229bd8aeaf7bb58df0f510e89f4c32d

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff
